browser-build.config.js

```diff
@@ -0,0 +1,51 @@
+const ExtractTextPlugin = require("extract-text-webpack-plugin");
+ module.exports = {
+  entry: "./src/index.js",
+  output: {
+      filename: "geostyler.js",
+      path: __dirname + "/browser",
+      library: 'GeoStyler'
+  },
+   // Enable sourcemaps for debugging webpack's output.
+  devtool: "source-map",
```

Comment:
I'm not sure if we really want source maps in production as they would bloat the output/build size.